### PR TITLE
Back to home once again

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds some basic content for the Tau ceti system.

### CODE REVIEW

File: code/modules/maps/planet_types/lore/tauceti.dm [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/update_icon()
+	return
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/generate_ground_survey_result()
+	ground_survey_result = "" // so it does not get randomly generated survey results
+
+// --------------------------------- Luthien
+
+/obj/effect/overmap/visitable/sector/exoplanet/desert/luthien
+	name = "Caprice"
```
Comment (GMR25) [inline code/modules/maps/planet_types/lore/tauceti.dm:21]:
```suggestion
	name = "Luthien"
```

File: code/modules/maps/planet_types/lore/tauceti.dm [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	icon_state = "globe1"
+	color = "#cf1020"
+	generated_name = FALSE
+	ring_chance = 0
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/update_icon()
+	return
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/generate_ground_survey_result()
+	ground_survey_result = "" // so it does not get randomly generated survey results
```
Comment (DreamySkrell) [inline code/modules/maps/planet_types/lore/tauceti.dm:16]: All new planets should have survey results defined for survey probes.

**Review by DreamySkrell — DISMISSED** [review]
as above

File: code/modules/maps/planet_types/lore/tauceti.dm (hunk 2 above)
Comment (DreamySkrell) [reply code/modules/maps/planet_types/lore/tauceti.dm:16]: I can help with these if it's not clear what should be put there.

Comment (DreamySkrell) [reply code/modules/maps/planet_types/lore/tauceti.dm]: this was not resolved - now they just use the randomly generated survey results, different every round

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**